### PR TITLE
feature/42-testing, Setup project for testing vue components

### LINKED ISSUE #42: Add support for testing vue components.
**DoD**
- A developer can create tests inside the "__tests__" folder and run the tests using the "yarn test:unit" command. (Note: must be inside the frontend directory for this command to work.) 

**Additional context**
Why? We need a way to test our components to prevent upcoming features from accidentally messing up our code!

### PR DESCRIPTION
**Pull request by TE4-Daniel-Kull**
# Pull Request Template

## What issue did you implement or fix?
Fixed #42

## Summary
- Added support for adding tests for vue components.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

## How can this implementation be improved?
- Tests could be added for existing modules, components and views.

### CODE REVIEW

File: frontend/src/components/ExampleInputComponent.vue [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<template>
```
Comment (itggot-linus-styren) [inline frontend/src/components/ExampleInputComponent.vue]: Is there a need include this test component? Perhaps it should be included in a wiki page for this project as a example on how to test.

File: frontend/src/components/ExampleTestComponent.vue [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<template>
```
Comment (itggot-linus-styren) [inline frontend/src/components/ExampleTestComponent.vue]: See above comment.

File: frontend/src/components/__tests__/ExampleTestComponent.spec.ts [hunk 3]
```diff
@@ -0,0 +1,49 @@
+import {mount, shallowMount} from '@vue/test-utils'
```
Comment (itggot-linus-styren) [inline frontend/src/components/__tests__/ExampleTestComponent.spec.ts]: Perhaps you could write a test for a existing component.

File: frontend/src/components/ExampleInputComponent.vue (hunk 1 above)
Comment (TE4-Daniel-Kull) [reply frontend/src/components/ExampleInputComponent.vue]: me liek thx!
KThnks bye

Comment (TE4-Daniel-Kull) [reply frontend/src/components/ExampleInputComponent.vue]: I'm a bit too lazy to do it right now, but I created an issue for it! :D

File: frontend/src/components/__tests__/ExampleTestComponent.spec.ts (hunk 3 above)
Comment (TE4-Daniel-Kull) [reply frontend/src/components/__tests__/ExampleTestComponent.spec.ts]: We've just introduced the concept of tdd so we are planing to do this in the near future when we've gotten a bit more used to the idea of tdd. :)

**Pull request merged**